.cirrus.yml

```diff
@@ -1127,6 +1153,12 @@ artifacts_task:
         - tar xjf repo.tbz
         - mv ./podman-remote-release-darwin_*.zip $CIRRUS_WORKING_DIR/
         - mv ./contrib/pkginstaller/out/podman-installer-macos-*.pkg $CIRRUS_WORKING_DIR/
+    #freebsd_binaries_script:
+    #    - mkdir -p /tmp/freebsd
+    #    - cd /tmp/freebsd
+    #    - $ARTCURL/FreeBSD%20Cross/repo/repo.tbz
+    #    - tar xjf repo.tbz
+    #    - mv ./podman-release-amd64.tar.gz $CIRRUS_WORKING_DIR/podman-release-freebsd-amd64.tar.gz
```

Comment:
Hrmmm, yeah, this can be tricky and fail for non-obvious reasons.  What was the error you got?  Let me guess, a "404" :grin: 

Do you think test-builds coming from CI will be useful to freebsd contributors?  I have no way of judging this, but I would offer that even the mainstream podman artifacts are likely of dubious utility.

Consider just clobbering this (and the `_script` steps that produce it).  It can always be added later if there is a need.

Reply:
The error was "Failed to upload artifacts: rpc error: code = FailedPrecondition desc = Artifacts size is too big!" and some googling seemed to suggest there is a 1Gb limit free tier for artifacts. The 'run contents' tab for a failing build also seems to put the total size over 1Gb (https://api.cirrus-ci.com/v1/task/6058945505132544/logs/contents.log) and a successful build without the freebsd binaries is just under (https://api.cirrus-ci.com/v1/task/5881770722197504/logs/contents.log).

I'm not sure how useful these builds are - I expect most people to use the packages coming from the pkg.freebsd.org infrastructure which are build from source and signed. I'll remove this section for now

Reply:
Indeed. There is some low-hanging fruit there - leaving the podman, podman-remote etc. binaries packed into a tarball instead of extracted and uncompressed, not having two copies of the windows MSI etc.